technic/init.lua

```diff
@@ -5,6 +5,7 @@
 local load_start = os.clock()
 
 technic = rawget(_G, "technic") or {}
+technic.plus = true
```

Comment:
Do we want to include actual version right now with this PR?
I'd say no, instead use boolean for now and think about how to best represent version.
Open issue or pull request for actual version, add to v2.0 milestone, possibly take a look at existing things that provide version information through API and see if there's clear upsides / downside with different approaches.

No matter what will be decided for format it will still be true as truth value so `true` should be good until something else is decided.

Reply:
Yeah, I think boolean is enough for now 👍